dla_cnn/desi/DesiMock.py

```diff
@@ -1,5 +1,7 @@
 from astropy.io import fits
 import numpy as np
+from data_model.Sightline import Sightline
```

Comment:
This should be

`from dla_cnn.data_model.Sightline import Sightline`

and the same for `Dla`

Reply:
Ok!I will change it.